console.warn("TEST")
console.error("TEST AGAIN")
console.log("FAILED TEST")

const first = () => {
    return ""
}